modules/oadp-creating-restore-cr.adoc

```diff
@@ -36,10 +36,11 @@ spec:
   - backups.velero.io
   - restores.velero.io
   - resticrepositories.velero.io
-  restorePVs: true
+  restorePVs: true <3>
 ----
 <1> Name of the `Backup` CR.
 <2> Optional. Specify an array of resources to include in the restore process. Resources might be shortcuts (for example, 'po' for 'pods') or fully-qualified. If unspecified, all resources are included.
+<3> Optional: `restorePVs` can be set to `false` in order to turn off restore of `PersistentVolumes` from `VolumeSnapshot` of Container Storage Interface (CSI) snapshots, or from native snapshots when `VolumeSnaphsotLocation` is configured.
```

Comment:
Suggestion:

```suggestion
<3> Optional: The `restorePVs` field can be set to `false` in order to turn off restore of `PersistentVolumes` from `VolumeSnapshot` of Container Storage Interface (CSI) snapshots, or from native snapshots when `VolumeSnaphsotLocation` is configured.
```

(If you implement this change, please avoid clicking "accept suggestion" and instead make this change locally)